libkbfs/data_types.go

```diff
@@ -814,12 +819,24 @@ const (
 	InitSingleOp
 )
 
+// Mode returns the mode absent any mode flags.
+func (im InitMode) Mode() InitMode {
+	return im & InitModeMask
+}
+
+// HasFlags returns whether all the specified flags are set.
+func (im InitMode) HasFlags(flags InitMode) bool {
+	return im&flags > 0
+}
+
 func (im InitMode) String() string {
 	switch im {
```

Comment:
Should this be `im.Mode()` now that it can contain flags?

Reply:
Nice catch! Yes it should.